data/strings/sound.txt

```diff
@@ -426,8 +481,93 @@
     tr = çıkın.
     uk = З'їзд.
     vi = Lối thoát.
-    zh-Hans = 出口.
-    zh-Hant = 出口.
+    zh-Hans = 驶出。
+    zh-Hant = 出口。
+
+  [onto]
+    comment = Used in dist_direction_onto_street in position 3 unless take_exit_number is specified. For languages that don't have an "onto" word, (e.g. eu, ja, ko), the value should be empty.
+    en = onto
+    ar = إلى
+    be = на
+    cs = na
+    da = ad
+    de = auf
+    el = στην
+    es = hacia
+    es-MX = hacia
+    eu =
+    fa = ﺐﻫ
+    fi = tielle
+    fr = sur
+    hi = पर
+    hr = na
+    hu = a
+    id = ke
+    it = sulla
+    ja = 
+    ko =
+    mr = वर
+    nb = inn på
+    nl = naar
+    pl = na
+    pt = para
+    pt-BR = para
+    ro = pe
+    ru = на
+    sk = na
+    sv = in på
+    sw = kwenye
+    th = เข้าสู่
+    tr = yönünde
+    uk = на
+    vi = vào
+    zh-Hans = 上
+    zh-Hant = 上
+
+  [take_exit_number]
+    comment = Used in dist_direction_onto_street instead of "Exit" when there's an exit number after. Replaces the "onto" string as well. (There is no take_exit_number_street since this string is only used when announcing street names anyway.)
+    en = Take exit
+    ar = خروج إلى
+    be = З'езд на
+    ca = Agafa la sortida
+    cs = výstup na
+    da = Afkørsel ad
+    de = Nehmen Sie die Ausfahrt auf
+    el = Εξοδος στην
+    es = Tome la salida
+    eu = Irteera
+    fa = خروجﺐﻫ
+    fi = Poistu tielle
+    fr = Sortie sur
+    hi = बाहर जाएं पर
+    hr = Izlaz na
+    hu = Hajtson ki a
+    id = Keluar ke
+    it = Uscita sulla
+    ja = 出口を出る
+    ko = 출구
+    mr = बाहेर पडा वर
+    nb = Ta avkjøringen inn på
+    nl = Verlaat naar
+    pl = Wybierz zjazd
+    pt = Siga pela saída
+    pt-BR = Pegue a saída
+    ro = Ieșire pe
+    ru = Съезд на
+    sk = Výjazd na
+    sv = Utgång in på
+    sw = Kuondoka kwenye
+    th = ทางออก เข้าสู่
+    tr = çıkın yönünde
+    uk = З'їзд на
+    vi = Lối thoát vào
```

Comment:
“Lối thoát” means “an escape”, and “vào” would introduce a street name, not an exit number.

```suggestion
    vi = Đi theo lối ra số
```

Reply:
@1ec5 all new strings that haven't had a human reviewer in the comments here are machine generated. You can see the list of language teams at the bottom and who's proposed changes; any and all help is definitely appreciated.

Reply:
Translation change request:
```
# data/strings/sound.txt
  [exit]
-     vi = Lối thoát.
+     vi = Đi theo lối ra
  [take_exit_number]
-     vi = Lối thoát vào
+     vi = Đi theo lối ra số
  [dist_direction_onto_street]
-     vi = %1$s %2$s %3$s %4$s
+     vi = undefined
```